third-party-drivers/phantomjs.rst

```diff
@@ -0,0 +1,85 @@
+PhantomJSDriver
```

Comment:
if it is small, you could also use github itself to host the documentation (write it in markdown in this case though, as their rST renderer is not as good as their markdown one) (putting the doc in the ``/docs`` folder of the repo for instance)